setuptools/command/editable_wheel.py

```diff
@@ -379,13 +380,13 @@ def _select_strategy(
 class EditableStrategy(Protocol):
     def __call__(
         self, wheel: WheelFile, files: list[str], mapping: Mapping[str, str]
-    ): ...
+    ) -> object: ...
     def __enter__(self) -> Self: ...
     def __exit__(
         self,
-        _exc_type: object,
-        _exc_value: object,
-        _traceback: object,
+        _exc_type: type[BaseException] | None,
+        _exc_value: BaseException | None,
+        _traceback: TracebackType | None,
```

Comment:
Yeah at one point I forgot I can't have a protocol or base classe declare its params as "unused" otherwise all subclasses/implementations must as well to respect the LSP.